<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixelcat Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Loaded Graphics</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>9</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Area</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1157</x>
      <y>9</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Layers</string>
    </property>
   </widget>
   <widget class="pce::graphicslist_widget" name="lw_tilesets">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>200</width>
      <height>353</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="pce::edit_area" name="w_edit_area" native="true">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>31</y>
      <width>936</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lw_layers">
    <property name="geometry">
     <rect>
      <x>1157</x>
      <y>29</y>
      <width>170</width>
      <height>353</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>416</y>
      <width>200</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>291</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QCheckBox" name="cb_showgrid">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>61</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Grid</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_grid_x">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>31</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>64</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_grid_y">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>31</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>64</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>32</y>
       <width>38</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>38</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid X:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_gridcolor">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>HHHHHH;#</string>
     </property>
     <property name="placeholderText">
      <string>Grid color (hex)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_reset_scale">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset scale</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_scale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>190</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>123</y>
       <width>12</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_add_layer">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>400</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Add Layer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_remove_layer">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>430</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Remove Layer(s)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>470</y>
      <width>170</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Layer Settings</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>pce::graphicslist_widget</class>
   <extends>QListWidget</extends>
   <header location="global">pce/graphicslist_widget.hpp</header>
  </customwidget>
  <customwidget>
   <class>pce::edit_area</class>
   <extends>QWidget</extends>
   <header location="global">pce/edit_area.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
